lib/aws/rails/action_mailbox/rspec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'aws/rails/action_mailbox/rspec/email'
+require 'aws/rails/action_mailbox/rspec/subscription_confirmation'
+require 'aws-sdk-sns'
+
+module Aws
+  module Rails
+    module ActionMailbox
+      # @api private
```

Comment:
These are public test methods that we are delivering, right? Should probably document each method here.

Reply:
they are documented in the readme, but I can add them here

Reply:
neat!